agent/csi/plugin/plugin_test.go

```diff
@@ -106,6 +107,28 @@ func TestNodePublishVolume(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestNodePublishVolumeWithoutStaging(t *testing.T) {
```

Comment:
Can we go ahead and make this a subtest with a staging bool and a callback to assert error/no error?